.github/workflows/build.yml

```diff
@@ -70,6 +70,8 @@ jobs:
           rm -rf output/bitwarden_event_logs/appserver/templates/base.html
           rm -rf output/bitwarden_event_logs/default/{restmap.conf,web.conf,bitwarden_event_logs_settings.conf}
           rm -rf output/bitwarden_event_logs/README/bitwarden_event_logs_settings.conf.spec
+          rm -rf output/bitwarden_event_logs/lib/charset_normalizer
+          mv output/bitwarden_event_logs/lib/linux/charset_normalizer output/bitwarden_event_logs/lib/charset_normalizer
```

Comment:
Can you provide some info about what this dependency is and how we use it?

Reply:
Also, does `package.sh` require a similar update? (Are there any differences between these scripts that require the copy/paste here?)

Reply:
added a bit more context in the objective.